javascript/javascript1/week3/script.js

```diff
@@ -0,0 +1,138 @@
+//Item array removal
+const names = [
+    "Peter",
+    "Ahmad",
+    "Yana",
+    "kristina",
+    "Rasmus",
+    "Samuel",
+    "katrine",
+    "Tala",
+  ]; 
+  const nameToRemove = "Ahmad";
+  for(let arrayIndex of names ){
+    if(nameToRemove === arrayIndex){
+        names.splice(arrayIndex ,1)    
+    }
+  }
+  console.log('Remaining array after removel:  '+names);
+
+  //When will we be there??
+  const travelInformation = {
+    speed: 50,
+    destinationDistance: 432,
+  };
+  function notThisFunctionName(){
+    return( travelInformation.destinationDistance/travelInformation.speed);
+  }
+  const travelTime = notThisFunctionName(travelInformation);
+  const timeinHhMm=timeConvertiontoHhMm( travelTime);
+  //refrece:how to covert decimal time to HH:mm  https://stackoverflow.com/questions/35460303/how-to-convert-decimal-hour-value-to-hhmmss/35460620
+  function timeConvertiontoHhMm(decimaltime){
+  let hrs = parseInt(Number(decimaltime));
+  let min = Math.round((Number(decimaltime)-hrs) * 60);
+  return(hrs+'hh:'+min+'mm');
+}
```

Comment:
Make sure you use consistent spaces. It makes code easier to read. Are you using Visual Studio Code? If you are, you can tell VS code to Format Document.

```suggestion
function timeConvertiontoHhMm(decimaltime) {
  let hrs = parseInt(Number(decimaltime));
  let min = Math.round((Number(decimaltime)-hrs) * 60);
  return hrs+'hh:'+min+'mm';
}
```